pymc/gp/gp.py

```diff
@@ -541,16 +562,17 @@ def predict(self, Xnew, point=None, diag=False, pred_noise=False, given=None):
             Default is `False`.
         given: dict
             Same as `conditional` method.
+        jitter: scalar
+            A small correction added to the diagonal of positive semi-definite
+            covariance matrices to ensure numerical stability.  For conditionals
+            the default value is 0.0.
         """
         if given is None:
             given = {}
+        mu, cov = self._predict_at(Xnew, diag, pred_noise, given, jitter)
+        return replace_with_values([mu, cov], replacements=point, model=model)
```

Comment:
@ricardoV94 So I was changing `replace_with_values` to take in replacements keyed by random variable instead of string, so `{a: 3}` instead of `{"a": 3}`.  I realized the reason I did it that was was because it takes as input `model.Point`, or the result of `pm.find_MAP` for instance.  I think these will have string keys, unless this has changed?

Reply:
That had not changed no

Reply:
okay cool.  if it changes to use RV keys in the future it'll be easy to update it here